www/core/nightlies/next_patch_extension.xml

```diff
@@ -82,10 +82,10 @@
 		<description>Joomla! CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>5.2.1-dev</version>
+		<version>5.2.2-dev</version>
```

Comment:
Should be 5.2.3-dev because when 5.2.2 is released, nightly builds should be 5.2.3-dev.

This applies to the nightlies.